docs/circuits.ipynb

```diff
@@ -731,7 +718,7 @@
     "id": "98a7293a6e4b"
    },
    "source": [
-    "Finally, the `mapped_circuit` method will return the circuit that a `CircuitOperation` represents after all repetitions and remappings have been applied. By default, this only \"unrolls\" a single layer of `CircuitOperation`s; to recursively unroll all layers, we can pass `deep=True` to this method."
+    "Finally, the `mapped_circuit` method will return the circuit that a `CircuitOperation` represents after all repetitions and remappings have been applied. By default, this only \"unrolls\" a single layer of `CircuitOperation`s: To recursively unroll all layers, you can pass `deep=True` to this method."
```

Comment:
I disagree with the `; to` -> `: To` conversion. The second clause doesn't explain the first, but rather introduces an alternate process. `. To` seems reasonable if greater separation is desired.